VoxelSniperCore/src/main/java/com/github/kevindagame/brush/polymorphic/operation/BlendOperation.kt

```diff
@@ -2,15 +2,19 @@ package com.github.kevindagame.brush.polymorphic.operation
 
 import com.github.kevindagame.brush.polymorphic.PolyBrush
 import com.github.kevindagame.brush.polymorphic.PolyOperationType
+import com.github.kevindagame.brush.polymorphic.property.PolyPropertiesEnum
+import com.github.kevindagame.snipe.SnipeAction
 import com.github.kevindagame.voxelsniper.material.VoxelMaterial
 
 class BlendOperation: PolyOperation(listOf(PolyOperationType.BLOCK)) {
     override fun apply(
         brushSize: Int,
         brush: PolyBrush,
+        snipeAction: SnipeAction,
         excludeAir: Boolean,
         excludeWater: Boolean
     ): Array<Array<Array<VoxelMaterial>>> {
+        val doExcludeAir = snipeAction == SnipeAction.GUNPOWDER
```

Comment:
Do we than still need the excludeAir argument?

Reply:
Perhabs not, but for now I'll leave it there

Reply:
sure